packages/api-db-mongodb/src/db/comment.ts

```diff
@@ -0,0 +1,184 @@
+import {
+  DBCommentAdapter,
+  CreateCommentArgs,
+  Comment,
+  GetCommentsArgs,
+  ConnectionResult,
+  CommentState,
+  LimitType,
+  InputCursorType,
+  CommentSort,
+  SortOrder,
+  PublicComment,
+  CommentAuthorType
+} from '@wepublish/api'
+
+import {Collection, Db, FilterQuery, MongoCountPreferences} from 'mongodb'
+import {Cursor} from './cursor'
+import {MaxResultsPerPage} from './defaults'
+
+import {CollectionName, DBComment} from './schema'
+
+function commentSortFieldForSort(sort: CommentSort) {
+  switch (sort) {
+    case CommentSort.CreatedAt:
+      return 'createdAt'
+
+    case CommentSort.ModifiedAt:
+      return 'modifiedAt'
+  }
+}
+
+function commentDateForSort(comment: DBComment, sort: CommentSort): Date {
+  switch (sort) {
+    case CommentSort.CreatedAt:
+      return comment.createdAt
+
+    case CommentSort.ModifiedAt:
+      return comment.modifiedAt
+  }
+}
+
+export class MongoDBCommentAdapter implements DBCommentAdapter {
+  private comments: Collection<DBComment>
+  private locale: string
+
+  constructor(db: Db, locale: string) {
+    this.comments = db.collection(CollectionName.Comments)
+    this.locale = locale
+  }
+
+  async createComment({input}: CreateCommentArgs): Promise<PublicComment> {
```

Comment:
Everything is good here. I'm just not happy with the name `createComment`. For all the other database adapter we use `createXYZ` for the method to create something from the editor. Maybe `addPublicComment`  is a better name here.

Reply:
Better naming :+1:  , fixed